yellowbrick/text/tsne.py

```diff
@@ -177,6 +177,8 @@ def __init__(self, ax=None, decompose='svd', decompose_by=50, labels=None,
         self.colors = colors
         self.colormap = colormap
         self.random_state = random_state
+        self._size = kwargs.pop('size', None)
+        self._title = kwargs.pop('title', None)
```

Comment:
Does this work if you set `self.title = kwargs.pop('title', None)` instead? The `Visualizer.title` is not a `property` but it is an attribute that's used to determine the default vs. user title if specified. (See comment below).